app/db/index.js

```diff
@@ -21,7 +21,7 @@ async function postOrderDb(reqBody) {
       supplier: reqBody.supplierAddress,
       required_by: reqBody.requiredBy,
       items: reqBody.items,
-      buyer: reqBody.buyerAddress,
```

Comment:
why is this key changed?

Reply:
In order table schema the key is purchaser not buyer